### DISCUSSION TITLE
C++ wrap代码的生成工具，为何在puerts_unreal_demo工程下面有，在puerts工程中没有呢？

### DISCUSSION THREAD

**Discussion by zhaojunmeng** [Tencent/puerts#1134] [Q&A] [answered]
下面这个工具，没有在puerts工程中找到
https://github.com/chexiongsheng/puerts_unreal_demo/tree/master/Plugins/Puerts/Generator

**Comment by chexiongsheng** [comment, marked as answer]
废弃了。
前期探索的路之一，现在只保留用作几个结构体的wrap代码生成。
非反射api建议用静态模板方式。

**Reply by zhaojunmeng** [reply]
感谢解答
